src/swsssdk/port_util.py

```diff
@@ -38,6 +38,9 @@ def get_interface_oid_map(db):
     """
     db.connect('COUNTERS_DB')
     if_name_map = db.get_all('COUNTERS_DB', 'COUNTERS_PORT_NAME_MAP', blocking=True)
+    oid_pfx = len("oid:0x")
+    if_name_map = {if_name: sai_oid[oid_pfx:] for if_name, sai_oid in if_name_map.items()}
```

Comment:
>sai_oid [](start = 28, length = 7)

This change will bring dependency on schema version. Suggest to make it backward compatible, so we can still update snmp docker in old switches.

Reply:
there could be other changes in the counters_db or other new data in the database that is needed by new snmp. It will be very difficult to test all cases. Not sure we really need to have this as a goal.